fluentd/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM fluent/fluentd:v1.1.3-onbuild
+MAINTAINER Petr Kotas <petr.kotas@gmail.com>
+
+EXPOSE 24224
```

Comment:
I see `fluent/fluentd:v1.1.3-onbuild` already expose this port
https://github.com/fluent/fluentd-docker-image/blob/94ab981b056210cdd331413215724445faf6ee71/v1.1/alpine-onbuild/Dockerfile#L54
so no need for separate custom `Dockerfile`